api/vispy.app.tests.rst

```diff
@@ -0,0 +1,70 @@
+vispy.app.tests package
+=======================
+
+Submodules
+----------
+
+vispy.app.tests.test\_app module
+--------------------------------
+
+.. automodule:: vispy.app.tests.test_app
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+vispy.app.tests.test\_backends module
+-------------------------------------
+
+.. automodule:: vispy.app.tests.test_backends
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+vispy.app.tests.test\_context module
```

Comment:
This seems like a lot of boilerplate stuff. No way to have this done automatically via `autodoc` / `autosummary`?

Reply:
(On the fly I mean, so no need to add it to the repo, just have it created during the sphinx build)

Reply:
Ah at first I was like "this is `sphinx-apidocs`, does he think I wrote all this by hand?".

I've thought about that. Somehow even though I figured travis would update these automatically I didn't make the obvious connection that that would mean not storing them in the repository at all. That makes sense though.

It might be easiest to make `conf.py` generate this on the fly, thoughts? I figure it would be most compatible and most "automatic" for people running things manually.

Reply:
Haha there's a package for doing it: https://pypi.org/project/sphinxcontrib-apidoc/#description

Reply:
Done. Next commit in the main repository will include setup.py and travis updates.